### DISCUSSION TITLE
Someone is making money with this free open source project

### DISCUSSION THREAD

**Discussion by siasbaily** [teslamate-org/teslamate#3786] [General]
https://www.teslamate.cloud/
This cannot guarantee data security and privacy. Should he be stopped?

**Comment by cwanja** [comment]
Why? Software is provided as is and they are welcome to spin up a service.

**Comment by brianmay** [comment]
The license - MIT License - is here: https://github.com/teslamate-org/teslamate/blob/master/LICENSE

There are no restrictions on commercial use. There is no requirement to distribute changes (unlike say the GPL).

Security - particularly of the Tesla's credentials are of concern. I would want to get more information here before I used their service, as to what safe guards they employ to ensure that neither my Tesla username/password or my Tesla token is leaked. I could make assumptions based on the software, but better not to when security is concerned. However I am not their target audience anyway.

**Comment by bmerciergallay** [comment]
> https://www.teslamate.cloud/ This cannot guarantee data security and privacy. Should he be stopped?

Absolutely not, Open-source has never been restricted to free-use unless it has a specific license article, you can absolutely charge for distributing or packaging an open source project especially when providing it as a service with your infrastructure which has a cost.

Most commercial softwares use at least some open-source libraries or software, as long as you comply with the license (gnu gpl or else) you can charge for your software using open-source products
